src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java

```diff
@@ -60,6 +67,41 @@ public String viewDashboardAll(
         return "dashboard/all_items";
     }
 
+    @RequestMapping("/expired")
+    public String viewExpiredItems(
+            Model category,
+            Model categoryList,
+            Model item,
+            Model itemList,
+            @AuthenticationPrincipal CustomUserDetails userDetails) {
+        final String endDate = simpleDate.format(new Date());
+        item.addAttribute("item", new Item());
+        category.addAttribute("category", new Category());
+        String userEmail = userDetails.getUsername();
+        User user = userService.getByEmail(userEmail);
+        itemList.addAttribute("listItems", itemService.getByUserIdBefore(user.getId(), endDate));
+        categoryList.addAttribute("listCategories", categoryService.getByUserId(user.getId()));
+        return "dashboard/expired";
+    }
+
+    @RequestMapping("/expires_soon")
+    public String viewExpiresSoonItems(
+            Model category,
+            Model categoryList,
+            Model item,
+            Model itemList,
+            @AuthenticationPrincipal CustomUserDetails userDetails) {
+        final String startDate = simpleDate.format(new Date());
+        final String endDate = LocalDate.parse(simpleDate.format(new Date())).plusDays(5).toString();
```

Comment:
you shouldn't hardcode this number 5. you should have an app config, that tell you this kind of information. do you understand what I mean by app config?

Reply:
Hmm, no, I didn't understand why I need this if for now, the users can't change this number of days, and also, isn't very clear what you want to say with this app config.  Please tell me more, if you believe this is a must in these conditions. Thank you!

Reply:
something like a centralized configuration menu for admins. to set thing like this

Reply:
it's a table in the database that contains different info for some global settings in your app

Reply:
Ok. Now I think I understood. I will create an issue for this task. Thank you.